### ISSUE TITLE
melpa checkout shouldn't look for `el` anymore

### ISSUE THREAD

**Issue by steckerhalter** [quelpa/quelpa#112]
```
Updating /home/user/.emacs.d/quelpa/melpa
Quelpa: failed to checkout melpa git repo: `No matching file(s) found in /home/user/.emacs.d/quelpa/melpa: (*.el *.el.in dir *.info *.texi *.texinfo doc/dir doc/*.info doc/*.texi doc/*.texinfo (:exclude .dir-locals.el test.el tests.el *-test.el *-tests.el))'
Fetcher: github
Source: quelpa/quelpa
```

**Comment by vyp** [comment]
When no `:files` is given in the recipe, `:files` defaults to `(*.el *.el.in dir *.info *.texi *.texinfo doc/dir doc/*.info doc/*.texi doc/*.texinfo (:exclude .dir-locals.el test.el tests.el *-test.el *-tests.el))` as from the error message.

So if we just want to checkout melpa for its recipes, maybe we could add `:files ("recipes")` to the config in the call to `package-build--checkout-git` from `quelpa-checkout-melpa`, or otherwise I think we can probably just put `:files ("*")`.

**Comment by steckerhalter** [comment]
Right, thanks, I guess `*` should do then because it doesn't really matter here, it's just for updating the git repo.

**Comment by steckerhalter** [comment]
fixed by #113
